hcl2template/types.build.provisioners.go

```diff
@@ -80,7 +80,7 @@ func (p *Parser) decodeProvisioner(block *hcl.Block) (*ProvisionerBlock, hcl.Dia
 		Except      []string `hcl:"except,optional"`
 		Rest        hcl.Body `hcl:",remain"`
 	}
-	diags := gohcl.DecodeBody(block.Body, nil, &b)
+	diags := gohcl.DecodeBody(block.Body, cfg.EvalContext(nil), &b)
```

Comment:
this is to decode variables used as value like: 
```
max_retries = var.max_retries
```
if we don't decode at this point this kind of usage doesn't work